src/js/ia/Helpers.js

```diff
@@ -129,6 +129,16 @@
         }
     });
 
+    // Check if val1 matches val2 using a regex
+    Handlebars.registerHelper('match', function(val1, val2, options) {
+       val2 = new RegExp(val2);
```

Comment:
Nope, it can also be "success: [some other stuff]", it's happened. I'll have to change the sidebar code as well in another PR